internal/install/execution/platform_link_generator.go

```diff
@@ -39,6 +45,10 @@ func (g *PlatformLinkGenerator) GenerateEntityLink(entityGUID string) string {
 // also provided in the nerdstorage document. This provides the user two options
 // to see their data - click from the CLI output or from the frontend.
 func (g *PlatformLinkGenerator) GenerateRedirectURL(status InstallStatus) string {
+	if status.HasFailedRecipes || status.HasCanceledRecipes {
+		return g.GenerateExplorerLink("")
```

Comment:
@sanderblue do we need to send any filter, hostname in the URL? if not  the URL should be simpler without the `platform[filters]=[hostname]` parts.
Maybe we should have a different method specific to this use case.

Also, wouldn't we want to have a same URL for success? We essentially want the user to go back to the NR1 Installation Plan page regardless of success/failure/cancel?

Reply:
We need the `platform[filters]` because of the `successLinkConfig` parameter in recipes.

Regarding when we add the referrer param: I was asking myself the same question. I think we can append this param in all situation and let the UI decide what it does with it.

Reply:
Making some updates. Will re-request review tomorrow.